<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_ml_config</class>
 <widget class="QWidget" name="widget_ml_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>1078</width>
     <height>804</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>23</y>
      <width>122</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Twitter dataset:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_file">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>23</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>open</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/open.png</normaloff>../assets/open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_clustering">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>901</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Clustering</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="check_use_minibatch_clustering">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>193</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>use minibatch k-means</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>98</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>k:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_clusters"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_clusterize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>clusterize data</string>
     </property>
     <property name="text">
      <string>clusterize</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>221</width>
       <height>106</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>features:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_features_clustering">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>done in:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_donein_clustering">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>silhuette coeficient:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_silhuette">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_save_clustering_config">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>33</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>save clustering config</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/save.png</normaloff>../assets/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>29</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>42</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>vectorizer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_vectorizer_clustering"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>195</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlayout_checkboxes">
      <item>
       <widget class="QCheckBox" name="check_use_stemming_clustering">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>use stemming</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_use_idf_clustering">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>use idf</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_binary_clustering">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>binary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_remove_stopwords_clustering">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>remove stopwords</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>192</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ngram range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_ngram_range_clustering"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>max df:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_max_df_clustering"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>min df:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_min_df_clustering"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>max features: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_max_features_clustering"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>170</y>
       <width>260</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="check_perform_lsa_clustering">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>perform lsa</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>number of components:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_n_components_clustering"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>explained variance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_explained_variance">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>24</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Percentage of stopwords:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_percentage_stopwords">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>24</y>
      <width>174</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lexical diversity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_lexical_diversity">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
